.github/workflows/release.yaml

```diff
@@ -12,56 +12,92 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-name: Release workflow
+name: Github Release and PyPi Publish
 
 on:
-  release:
-    types: [published, edited]
+  workflow_dispatch:
+  push:
+   tags:
+     - "v*.*.*"
 
 jobs:
-  release-package:
-    name: Generate package binaries
+  build:
+    name: Build distribution 📦
```

Comment:
can you change ubuntu version to 22.04

Reply:
Any particular reason? We use ubuntu-latest in quite many velocitas repos, and it is also what is currently used in https://github.com/eclipse-velocitas/vehicle-app-python-sdk/blob/main/.github/workflows/release.yaml?

Reply:
just to be on the safe side :)